sdk/keyvault/azkeys/autorest.md

```diff
@@ -8,13 +8,24 @@ version: "^3.0.0"
 input-file:
 - https://github.com/Azure/azure-rest-api-specs/blob/e2ef44b87405b412403ccb005bfb3975411adf60/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/keys.json
 license-header: MICROSOFT_MIT_NO_VERSION
-clear-output-folder: true
+clear-output-folder: false
 output-folder: internal/generated
-module: azkeys
 openapi-type: "data-plane"
 security: "AADToken"
 security-scopes:  "https://vault.azure.net/.default"
-use: "@autorest/go@4.0.0-preview.38"
+use: "@autorest/go@4.0.0-preview.41"
```

Comment:
FYI I released `preview.42` this morning that contains the change to omit the `moduleName` and `moduleVersion` constants.